src/bournemouth/chat_utils.py

```
@@ -7,10 +7,10 @@
 import typing
 
 import falcon
-from msgspec import Struct
+import msgspec
```

Pylint (3.3.7)

[error] Unable to import 'msgspec'

(E0401)